content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -363,6 +363,9 @@ Yet another example would be to first export the `ETCDCTL_API` environment varia
 export ETCDCTL_API=3
 etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
```

Comment:
Should the context for this line change? It might not be obvious to readers that this is a different tool, not just a different environment variable.

Reply:
@sftim You are right. As per your comments, I changed the context of the comment for the `etcdctl` tool. Also changed the position of the deprecation note to be before the command.
Please take a look.